scripts/build_powerplants.py

```diff
@@ -253,16 +253,6 @@ def add_custom_powerplants(ppl, inputs, config):
         return add_ppls
 
 
-def replace_natural_gas_technology(df):
-    mapping = {"Steam Turbine": "OCGT", "Combustion Engine": "OCGT"}
-    tech = df.Technology.replace(mapping).fillna("OCGT")
-    return df.Technology.where(df.Fueltype != "Natural Gas", tech)
-
-
-def replace_natural_gas_fueltype(df):
-    return df.Fueltype.where(df.Fueltype != "Natural Gas", df.Technology)
```

Comment:
"if Fueltype is not "Natural Gas" then set by default df.Technology" makes no sense to me. Probably here is the logical error... 

df.Fueltype.where(df.Technology = "OCGT", "Natural Gas") or something like that... Like if the technology field is OCGT make the fueltype natural gas